src/viur/core/bones/string.py

```diff
@@ -123,6 +126,10 @@ def isInvalid(self, value):
             return "Maximum length exceeded"
         if self.min_length is not None and len(value) < self.min_length:
             return "Minimum length not reached"
+
+        if self.charset:
+            if not all([char in self.charset for char in value]):
```

Comment:
This might by a little bit faster, because this will fail fast as generator with `any`.
```suggestion
            if any(char not in self.charset for char in value):
```